src/modules/tracker.js

```diff
@@ -1357,6 +1357,7 @@ class Tracker {
    * @param {number} [networkParameters.timeout] - Request timeout (in milliseconds)
    * @param {string} [parameters.slCampaignId] - Pass campaign id of sponsored listing
    * @param {string} [parameters.slCampaignOwner] - Pass campaign owner of sponsored listing
+   * @param {string[]|string|number} [parameters.seedItemIds] - Item ID(s) to be used as seed
```

Comment:
should we keep this as `string[]|string` to stay consistent with the results fn?

Reply:
Haha. I was thinking of having it just be `string[]` as well when I first looked over things, but makes sense to leave it to keep things consistent with the other event